plugins/resources/replication/irods_repl_rebalance.cpp

```diff
@@ -199,12 +200,13 @@ namespace {
         genquery_inp_wrapped.get().maxRows = _batch_size;
 
         std::stringstream cond_ss;
+        cond_ss << "IN (";
         for (auto& bun : _bundles) {
             for (auto id : bun) {
-                cond_ss << "= '" << id << "' || ";
+                cond_ss << "'" << id << "',";
```

Comment:
consider fmtlib.

there is nothing wrong with this form though.

Reply:
fmt, done